package.json

```diff
@@ -24,28 +28,34 @@
     "url": "https://github.com/onefinestay/react-daterange-picker"
   },
   "peerDependencies": {
-    "react": ">=0.12.0"
+    "react": ">=0.14.0"
   },
   "dependencies": {
     "calendar": "^0.1.0",
     "classnames": "^2.1.1",
     "immutable": "^3.7.2",
     "moment": "^2.10.6",
-    "moment-range": "^2.0.3"
+    "moment-range": "^2.0.3",
+    "react": "^0.14.0",
```

Comment:
We should remove `react` and `react-dom` from our dependencies, if they will be provided as peerDependencies